packages/flutter_app/analysis_options.yaml

```diff
@@ -9,6 +9,6 @@ custom_lint:
   rules:
     # TODO(riscait): Enable custom lint rules for altive_lints.
     # https://github.com/altive/flutter_app_template/issues/461
-    - avoid_hardcoded_color: false
+    - avoid_hardcoded_color: true
```

Comment:
![badge](https://img.shields.io/badge/review-imo-important)
Remove the false specification instead of the true specification as it is enabled by default.
```suggestion
```